### ISSUE TITLE
Broken syntax exporting `@pipelinex`, `@pipelinez`

### ISSUE THREAD

**Issue by c42f** [IBM/TSML.jl#206]
Developing the new Julia parser in JuliaSyntax.jl, I noticed TSML accidentally triggers a bug in the Julia reference parser:

https://github.com/IBM/TSML.jl/blob/26d5cd1b049ed47157ffe30ebbd77d2ddbc57357/src/TSML.jl#L27

```julia
julia> :(export @pipeline @pipelinex, @pipelinez)
:(export @pipeline)
```

So here you're exporting only `@pipeline` - it seems the others are ignored.

XRef https://github.com/JuliaLang/JuliaSyntax.jl/issues/134

**Comment by ppalmes** [comment]
this is a good catch, thanks.

**Comment by ppalmes** [comment]
this is now fixed in #207

**Comment by c42f** [comment]
Great!
